app/forms/candidate_interface/other_qualification_details_form.rb

```diff
@@ -7,14 +7,15 @@ class OtherQualificationDetailsForm
     attr_reader :next_step
     attr_accessor :editing, :id, :current_step,
                   :qualification_type, :other_uk_qualification_type, :non_uk_qualification_type,
-                  :subject, :grade, :predicted_grade, :award_year, :institution_country, :choice
+                  :subject, :grade, :predicted_grade, :award_year, :institution_country, :choice, :recruitment_cycle_year
 
     before_validation :sanitize_grade_where_required
 
     validates :qualification_type, presence: true, length: { maximum: ApplicationQualification::MAX_QUALIFICATION_TYPE_LENGTH }
     validates :non_uk_qualification_type, length: { maximum: ApplicationQualification::MAX_QUALIFICATION_TYPE_LENGTH }
 
-    validates :award_year, presence: true, numericality: { only_integer: true, in: 100.years.ago.year..RecruitmentCycle.next_year }
+    validates :award_year, presence: true, numericality: { only_integer: true }
+    validate :award_year_before_training_starts
```

Comment:
There is a problem with using something int he database in validations in tests, so simply changing the range to refer to `RecruitmentCycleTimetable.next_year` causes the whole suite to fail. You can simulate what happens by 
1. `bin/rails c`
2. RecruitmentCycleTimetable.destroy_all
3. exit
Then try to do `bin/rails c` again. You'll get this error: 
<img width="1829" alt="image" src="https://github.com/user-attachments/assets/2c653d33-c89e-4089-b9fd-f6926fbc6e40" />

So I've adopted the method of validating the degree end year, which is to use a `recruitment_cycle_year` attribute on the form.